build-support-windows/Dockerfile-consul-dev-windows

```diff
@@ -25,4 +33,4 @@ EXPOSE 19000 19001 19002 19003 19004
 EXPOSE 21000 21001 21002 21003 21004
 EXPOSE 5000 1234 2345
 
-RUN SETX /M path "%PATH%;C:\consul;C:\envoy;C:\fortio;C:\Program Files\Git\bin;C:\Program Files\OpenSSL-Win64\bin"
+RUN SETX /M path "%PATH%;C:\consul;C:\envoy;C:\fortio;C:\Program Files\Git\bin;C:\Program Files\OpenSSL-Win64\bin;C:\bats\bin\bats;C:\ProgramData\chocolatey\lib\jq\tools;"
```

Comment:
is the Path to bats binary OK?
I think that it's `C:\bats\bin` instead `C:\bats\bin\bats`

Reply:
The Bats Scripts should be working correctly in the unified container. When running the tests, it has to be executed directly in that container through a Docker exec.